extensions/spotify-player/src/api/playDiscoverWeekly.ts

```diff
@@ -0,0 +1,8 @@
+import { runAppleScript } from "@raycast/utils";
+import { buildScriptEnsuringSpotifyIsRunning } from "../helpers/applescript";
+
+export async function playDiscoverWeekly() {
+  const script = buildScriptEnsuringSpotifyIsRunning(`play track "spotify:playlist:37i9dQZEVXcQMiQXJGIMRO"`);
```

Comment:
Is this a hardcoded playlist for every user?

Reply:
Yes, but that id seems to open the Discover Weekly playlist for everyone. It's one of those dynamic ids, like how Spotify Wrapped playlists [can't be shared](https://community.spotify.com/t5/FAQs/Sharing-Your-Top-Songs-2022/ta-p/5453003) because everyone's Spotify Wrapped has the same id.